Code/IO/src/sitkShow.cxx

```diff
@@ -528,6 +590,8 @@ namespace itk
   std::string Macro = "";
   std::vector<std::string> CommandLine;
 
+  // is the imagej we're running a script or a binary?
+  bool ImageJScriptFlag = false;
```

Comment:
I believe this will be marked as an unused variable on apple and windows. Think you can declare it when it is initialized below.